samples/drivers/misc/ft800/src/main.c

```diff
@@ -23,63 +24,67 @@
 
 static volatile bool process_touch;
 
-static void touch_irq(void)
+static void touch_irq(const struct device *dev, void *user_data)
 {
+	(void)dev;
+	(void)user_data;
+
 	process_touch = true;
 }
 
 int main(void)
 {
```

Comment:
unrelated to your changes but this code should first perform an `if device_is_ready(...)` check